### DISCUSSION TITLE
How to access UI from outside container?

### DISCUSSION THREAD

**Discussion by Yolomir** [crawlab-team/crawlab#1176] [General]
I set up everything according to documentation but I can not access UI remotely. (SND docker-compose.yml)
Running locally is totally fine, so how do I set it up on my Ubuntu server so I can access through public ip:8080

Maybe this step could be added to documentation because I think there are many more who struggle with this

**Comment by tikazyq** [comment]
Is there any logs you can post here? Please try the command `docker-compose logs master` where `docker-compose.yml` is located.

**Comment by Yolomir** [comment]
docker-compose logs master:
[crawlab.txt](https://github.com/crawlab-team/crawlab/files/9437490/crawlab.txt)

modified docker file 
[modified_dockerfile.txt](https://github.com/crawlab-team/crawlab/files/9437491/modified_dockerfile.txt)

docker inspect container output:
[crawlab_custom_inspect.txt](https://github.com/crawlab-team/crawlab/files/9437492/crawlab_custom_inspect.txt)

**Reply by tikazyq** [reply]
Nothing odd here but I would suggest you start with official docker-compose.yml first and then try to make modifications. If you still have issues and would like more support, please join me on Wechat "tikazyq1".

**Reply by Yolomir** [reply]
Sorry it was totally my fault. My server was bad configured - discussion can be closed, sorry for bothering you

Crawlab is working like a charm! <3
